tests/e2e-leg-11/client-proxy/56-assert.yaml

```diff
@@ -0,0 +1,20 @@
+# (c) Copyright [2021-2024] Open Text.
```

Comment:
I remember you said the pod hostname cannot work after restarting, is that fixed? We can add one more test step: kill all vertica pods, and wait for them to be up, then use proxy to check if we can connect to the database.